react_native_fundamentals/presentation.html

```diff
@@ -2,43 +2,1027 @@
+            Initiating a project with React Native CLI can be <strong>time-consuming. </strong><br>This is mainly because of the <strong>complicated device configurations</strong>. You need to set the environment variables and ensure you have the right required SDKs for your application to function.<br>To build cross-platform mobile apps using the React Native CLI, you need both Andriod and IOS emulators. This means you need to have <strong>Andriod Studio and XCode</strong> to run your projects, which are available exclusively on the <strong>Mac.</strong> Sharing an app can be troublesome. This is because you need to build a <strong>whole APK or IPA file.</strong> <br>
+          </aside>
+        </section>
+
+        <section class="table-slide">
+          <h4 class="slide-name">React-native vs. Expo</h4>
+          <table class="fragment" data-fragment-index="2">
+            <thead>
+              <tr>
+                <th></th>
+                <th>React Native</th>
+                <th>Expo</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>Native modules</td>
+                <td>+</td>
+                <td>-</td>
+              </tr>
+              <tr>
+                <td>Hello World app size</td>
+                <td>~5 mb</td>
+                <td>~25 mb</td>
+              </tr>
+              <tr>
+                <td>Need Android Studio and XCode</td>
+                <td>+</td>
+                <td>-</td>
+              </tr>
+              <tr>
+                <td>App share</td>
+                <td>Harder (.apk or .ipa)</td>
+                <td>Easy (QR-code)</td>
+              </tr>
+            </tbody>
+          </table>
+          <aside class="notes">
+            Well, as you can see, the selection of the CLI is dependent on the requirements and feature set that is to be implemented.
+Given a comparison of the cons and props of both frameworks, I would suggest screening through the requirements and choosing which one suits your application most.
+          </aside>
+        </section>
+
+        <section class="picture-slide">
+          <h4 class="slide-name">Alternatives</h4>
+          <img src="assets/images/alternatives.png" alt="simple" />
+          <aside class="notes">
+            React Native is not the only one technology, that allow you to build mobile apps for both iOS and Android. You can also do it with Flutter and Ionic. 
+          </aside>
+        </section>
+
+        <section class="bullets-and-picture-slide">
+          <h4 class="slide-name">Flutter</h4>
+          <div>
+            <img src="assets/images/flutter_big.png" alt="simple" />
+            <div class="pros-cons-section">
+              <p class="big">
+                Pros:
```

Comment:
Maybe add "Type: Hybrid" or "Type: Native" accordingly

Reply:
I don't understand what you mean. Can you say with other words